guide/webpack.config.js

```diff
@@ -32,6 +33,15 @@ module.exports = validate({
                 include: RegExp(__dirname + "/src"),
                 loaders:["react-hot", "babel-loader"]
             },
+	    {
+		test: /\.txt$/,
+		loader: 'raw-loader',
+		include: path.resolve(__dirname, 'src/examples/raw-code'),
+	    },
+	    {
+		test: /\.md$/,
+		loader: 'raw-loader',
+	    },
```

Comment:
I am a supporting of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). Add support for loading `.txt` and `.md` when there is a _motivated_ need. One might argue that "we might need this" is how a configuration file goes from simple to complex.